inc/drivers/ST7735.h

```diff
@@ -58,6 +58,11 @@ class ST7735 : public CodalComponent
     // and doesn't support 12 bit color
     bool double16;
 
+    void beginCS() { if (cs) cs->setDigitalValue(0); }
+    void endCS() { if (cs) cs->setDigitalValue(1); }
+    void setCommand() { dc->setDigitalValue(0); }
```

Comment:
dc should also be if guarded just in case?

Reply:
no, the screen just won't work without dc, but it works fine without cs